cmd/node_agent/main.go

```diff
@@ -62,7 +62,12 @@ func main() {
 }
 
 func runNodeAgent() {
-	nodeAgent := na.NewNodeAgent(&naConfig)
+	nodeAgent, err := na.NewNodeAgent(&naConfig)
+	if err != nil {
+		glog.Fatal(err.Error())
+		os.Exit(1)
```

Comment:
you don't need this, Fatal will terminate the process.

Reply:
glog.Fatal() terminates the whole process, unit tests are not able to capture it
I changed the library code to return err and handle termination in the main.
ex. https://github.com/istio/auth/pull/279/files#diff-5bfee14e67b4a8fbd5f8b2d194d78324L33

If I misunderstood, please let me know.

Reply:
I think Tao meant you can remove line 68. glog.Fatal() can already terminate the process, so line 68 will never be reached.

Reply:
I mean, you don't need line 68, i.e., "os.Exit(1)" since the previous line already terminates the process.

Reply:
I got it. Thanks.